src/benchmarks/micro/libraries/System.Text.Encoding/Perf.Ascii.cs

```diff
@@ -0,0 +1,106 @@
+﻿using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Buffers;
+using System.Linq;
+
+namespace System.Text
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public class Perf_Ascii
+    {
+        [Params(
+            6, // non-vectorized code path
+            128)] // vectorized code path
```

Comment:
Should there be a size for Vector128/256 too?

The logic for Vec256 and Vec128 is actually the same, just different vector sizes.

Reply:
> Should there be a size for Vector128/256 too?

You mean the smallest size that gets vectorized? Since we cover both byte and char inputs it would require adding at least few more cases, which would increase the time it takes to run the benchmarks (it's 6.5 minute currently). I would prefer to avoid doing that.